jmx-metrics/src/main/resources/target-systems/hbase.groovy

```diff
@@ -0,0 +1,177 @@
+  "The age of the longest region in transition.", "ms",
+  "ritOldestAge", otel.&longValueCallback)
+
+def beanRegionServerServer = otel.mbean("Hadoop:service=HBase,name=RegionServer,sub=Server")
+otel.instrument(beanRegionServerServer, "hbase.region_server.region.count",
+  "The number of regions hosted by the region server.", "{regions}",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "regionCount", otel.&longUpDownCounterCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.disk.store_file.count",
+  "The number of store files on disk currently managed by the region server.", "{files}",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "storeFileCount", otel.&longUpDownCounterCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.disk.store_file.size",
+  "Aggregate size of the store files on disk.", "By",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "storeFileSize", otel.&longUpDownCounterCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.write_ahead_log.count",
+  "The number of write ahead logs not yet archived.", "{logs}",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "hlogFileCount", otel.&longUpDownCounterCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.request.count",
+  "The number of requests received.", "{requests}",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  ["writeRequestCount":["state" : {"write"}], "readRequestCount": ["state" : {"read"}]],
+  otel.&longUpDownCounterCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.queue.length",
+  "The number of RPC handlers actively servicing requests.", "{handlers}",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  ["flushQueueLength":["state" : {"flush"}], "compactionQueueLength": ["state" : {"compaction"}]],
+  otel.&longUpDownCounterCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.blocked_update.time",
+  "Amount of time updates have been blocked so the memstore can be flushed.", "ms",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "updatesBlockedTime", otel.&longValueCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.block_cache.operation.count",
+  "Number of block cache hits/misses.", "{operations}",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  ["blockCacheMissCount":["state" : {"miss"}], "blockCacheHitCount": ["state" : {"hit"}]],
+  otel.&longValueCallback)
+otel.instrument(beanRegionServerServer, "hbase.region_server.files.local",
+  "Percent of store file data that can be read from the local.", "%",
+  ["region_server" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "percentFilesLocal", otel.&doubleValueCallback)
+
+otel.instrument(beanRegionServerServer, "hbase.region_server.operation.append.latency.p99",
```

Comment:
Would adding min, median, and/or mean be helpful for these operations?

Reply:
Added